tests/test_win_prob.py

```diff
@@ -0,0 +1,49 @@
+"""Tests Win Probability module."""
+
+import pytest
+
+from awpy import Demo
+from awpy.stats import win_probability
+
+class TestWinProbability:
+    """Tests the win probability calculations.
+    
+    https://www.hltv.org/matches/2369248/natus-vincere-vs-virtuspro-pgl-cs2-major-copenhagen-2024-europe-rmr-closed-qualifier-a
+    """
+
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup method called before any tests are run.
+        Initializes the Demo object and random weights for testing.
+        """
+        cls.demo = Demo(file='tests/natus-vincere-vs-virtus-pro-m1-overpass.dem')  
+
+
+    @classmethod
+    def teardown_class(cls):
```

Comment:
In which case, we won't need a teardown method (nor a setup)

Reply:
removed teardown method and setup and replaced with pytest fixtures